modules/tls-profiles-ingress-configuring.adoc

```diff
@@ -0,0 +1,108 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-ingress-configuring_{context}"]
+= Configuring TLS security profiles for the Ingress controller
```

Comment:
Configuring TLS security profiles -> Configuring the TLS security profile

Reply:
Good catch. I took that from the existing ingress controller doc. This make more sense.